queries.sql

```diff
@@ -101,3 +101,75 @@ LEFT JOIN animals AS a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name AS animal_name
+FROM visits v
+JOIN animals a ON v.animal_id = a.id
+WHERE v.vet_id = 1
+ORDER BY v.visit_date DESC
+LIMIT 1;
```

Comment:
- [ ] Please add a screenshot for this query result as per the requirement. I suggest you label each of the queries for the next reviewer to find it easily.

Reply:
Hi @tugsem 
It's included